server/src/pay-gov-mocks/christmas-trees-payments.es6

```diff
@@ -23,25 +24,54 @@ payGov.router.options('*', middleware.setCorsHeaders, (req, res) => {
 });
 
 payGov.router.post('/mock-pay-gov', function(req, res) {
-  const startOnlineCollectionRequest =
-    req.body['S:Envelope']['S:Body'][0]['ns2:startOnlineCollection'][0]['startOnlineCollectionRequest'][0];
-  const permitId = startOnlineCollectionRequest.permitId[0];
-
-  var xmlResponse = `<?xml version="1.0" encoding="UTF-8"?>
-                    <S:Envelope xmlns:S="http://schemas.xmlsoap.org/soap/envelope/">
-                      <S:Header>
-                       <work:WorkContext xmlns:work="http://oracle.com/weblogic/soap/workarea/">
-                       </work:WorkContext>
-                      </S:Header>
-                      <S:Body>
-                        <ns2:startOnlineCollectionResponse xmlns:ns2="http://fms.treas.gov/services/tcsonline">
-                          <startOnlineCollectionResponse>
-                            <token>${permitId}</token>
-                          </startOnlineCollectionResponse>
-                        </ns2:startOnlineCollectionResponse>
-                      </S:Body>
-                    </S:Envelope>`;
-
+  
+  const requestBody =
+    req.body['S:Envelope']['S:Body'][0];
+  
+  // const startOnlineCollectionRequest =
+  //   req.body['S:Envelope']['S:Body'][0]['ns2:startOnlineCollection'][0]['startOnlineCollectionRequest'][0];
+  let xmlResponse = '';
+    
+  const token = uuid();
+  const paygovTrackingId = util.getRandomString(5).toUpperCase();
+  
+  if(requestBody['ns2:startOnlineCollection'] 
+    && requestBody['ns2:startOnlineCollection'][0]['startOnlineCollectionRequest'][0]) {
+      
+    xmlResponse = `<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
would it make sense to have a file of just xml pay.gov templates? seems potentially cleaner.

Reply:
I've added a task to pull those out into their own file in #168